### DISCUSSION TITLE
High GPU usage on windows

### DISCUSSION THREAD

**Discussion by Amejonah1200** [zed-industries/zed#18352] [Feature Requests]
![image](https://github.com/user-attachments/assets/ab15b800-6bc6-4308-83e0-7e0bb4799675)

Is there a reason why the GPU usage on Windows 11 is 30% up to 100% when doing basic tasks like scrolling, selecting and navigating?

I mean, I understand if it isn't much optimized but... why? It's not a video gam-

![image](https://github.com/user-attachments/assets/75348ad0-d901-4406-96c8-f040fedb1b35)

oh. But still.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
